packages/perf-benchmarks/rollup.config.js

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+
+import path from 'path';
+import glob from 'glob';
+import replace from '@rollup/plugin-replace';
+import syntheticShadow from './rollup-plugins/synthetic-shadow';
+import bestTachometerShim from './rollup-plugins/best-tachometer-shim';
+
+// Figure out all the packages we might be importing from perf-benchmarks-components
+// so that we can tell Rollup that those are `external`.
+const componentModules = glob
+    .sync(path.join(__dirname, '../perf-benchmarks-components/dist/**/*.js'))
+    .map((filename) => path.relative(path.join(__dirname, '..'), filename).replace(/^\.\//, ''));
+
+// these benchmarks are too fast (<~50ms) to give us consistent, usable results
+const disabledBenchmarks = [
+    'tablecmp-update-10th-1k',
+    'tablecmp-clear-1k',
+    'wc-clear-1k',
+    'wc-update-10th-1k',
+    'ss-slot-update-container-template',
+    'ss-slot-update-component-with-slot',
+    'table-append-1k',
+    'table-create-1k',
+    'table-update-10th-1k',
+    'table-clear-1k',
+];
```

Comment:
I think getting rid of the `1k`s probably makes sense, yeah. There may be cases where 10k and 1k disagree, but in those cases I'd prefer to rely on 10k since it's longer (therefore more reliable) and more likely to test peak performance (which is what microbenchmarks are good at anyway).

`clear-10k` sounds great as well.

Reply:
@pmdartus I looked into adding `clear-10k` tests (https://github.com/nolanlawson/lwc-1/commit/82f99d02c4e9f383b6d53aa519692fe682eae12b), but they take <50ms to run so I don't think it's worth it. We could make it `clear-100k` but then it would take a long time in the "before" part of the benchmark:

![Screen Shot 2021-07-28 at 11 39 51 AM](https://user-images.githubusercontent.com/283842/127378008-1c9b2b75-55fe-4236-9f70-296c0d7ba067.png)